src/home/Home.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react';
-import { SafeAreaView, Text } from 'react-native';
+import { StyleSheet, SafeAreaView, Text } from 'react-native';
+import RecentCard from '../home/components/RecentCard';
 
 const Home = ({ navigation }) => {
   return (
-    <SafeAreaView>
-      <Text>Home page here!</Text>
+    <SafeAreaView style={styles.background}>
+      <SafeAreaView style={{marginHorizontal: 10, flex: 1}}>
+        <Text style={styles.title}>Recently added activity cards</Text>
+
+        <SafeAreaView>
+          <RecentCard style={{flex: 3}}></RecentCard>
+          <RecentCard style={{flex: 3}}></RecentCard>
+          <RecentCard style={{flex: 3}}></RecentCard>
```

Comment:
You can make this into a StyleSheet variable and pass in: 
```javascript
          <RecentCard style={styles.card}></RecentCard>
```
Since they're all the same